01_network_sysapi/headers/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_library(Headers STATIC)
```

Comment:
You can list source files right here and later just include headers.
`add_library(Headers header.cpp)`